src/main/java/org/zalando/nakadi/config/PluginsConfig.java

```diff
@@ -0,0 +1,51 @@
+package org.zalando.nakadi.config;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.BeanCreationException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.DefaultResourceLoader;
+import org.zalando.nakadi.plugin.api.ApplicationService;
+import org.zalando.nakadi.plugin.api.ApplicationServiceFactory;
+import org.zalando.nakadi.plugin.api.SystemProperties;
+import org.zalando.nakadi.plugin.api.authz.AuthorizationService;
+import org.zalando.nakadi.plugin.api.authz.AuthorizationServiceFactory;
+
+@Configuration
+public class PluginsConfig {
```

Comment:
Probably, makes sense to move system props here as well, because they are exactly for these two beans